js-classes/inheritance-es6.js

```diff
@@ -0,0 +1,69 @@
+// Clasess in JS ES2015 CLASSES
+class NewCar {
+    constructor(name, model, year, color, maxSpeed, fuelCapacity = 60, fuelConsumption = 10) {
+        this.name = name;
+        this.model = model;
+        this.year = year;
+        this.color = color;
+        this.maxSpeed = maxSpeed;
+        this.fuelCapacity = fuelCapacity;
+        this.fuelConsumption = fuelConsumption;
+    }
+    getAge() {
+        return 'Current year - ' + this.year;
```

Comment:
It's better to use es6 syntax:
```suggestion
        return `Current year - ${this.year}`;
```